mysql-test/suite/rpl/include/rpl_shutdown_wait_slaves.inc

```diff
@@ -75,17 +75,26 @@ EOF
 --connection server_1
 DROP TABLE t1;
 
+# Slaves IO thread will receive the disconnect error when master was shutdown
+# so we are allowing error on start.
```

Comment:
This is a bit vague with the following behavior essentially starting twice:
```
--let rpl_allow_error=1
--source include/wait_for_slave_io_to_start.inc
--source include/start_slave.inc
```

I think it should be reworded to mention that the IO thread will automatically try to reconnect, and the error is allowed during this time, rather than "on start"